index.html

```diff
@@ -0,0 +1,342 @@
+        website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+    </div>
+    <hr class="line-3">
+    <div class="about-me-desk">
+      <div class="story-abt-me">
+        <h1 class="h31-desk">About me</h1>
+        <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+          website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.</p>
+        <a class="get-myresume button" href="">Get My Resume</a>
+      </div>
+    </div>
+
+    <ul class="lists">
+      <li class="Lan">Language
+        <ul class="lan-list">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </li>
+
+      <li class="Frame"> Framework
+        <ul class="Frame-list">
+          <li>React.js</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+        </ul>
+      </li>
+      <li class="Sk">Skills
+        <ul class="Sk-list">
+          <li>Database management</li>
+          <li>Version control</li>
+          <li>CLI</li>
+          <li>API design</li>
+          <li>Web development</li>
+        </ul>
+      </li>
+    </ul>
+  </section>
+  <form class="upper-form" action="https://formspree.io/f/xoqrrprd" method="post">
+    <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+    <input type="text" maxlength="30" required placeholder="Full Name" class="full-name-m" name="usrname">
+    <input type="email" required placeholder="Email" class="email-m" name="usremail">
+    <textarea maxlength="500" id="msg" placeholder="Enter text here" class="messages-m" name="usrmessages"></textarea>
+    <input type="submit" value="Get it touch" class="touch-m" name="usrsubmit">
+  </form>
+
+  <form class="upper-form-desk" action="https://formspree.io/f/xoqrrprd" method="post">
+    <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+    <div class="inputs-form">
+      <div class="form-name-desk">
+        <input type="text" maxlength="30" required placeholder="Name" class="full-name-desk" name="usrname">
+        <input type="text" maxlength="30" id="lnm" required placeholder="Last Name" class="last-name-desk"
+          name="usrname">
+      </div>
+      <input type="email" required placeholder="Email" class="email-desk" name="usremail">
+      <textarea maxlength="500" id="msg" placeholder="Hello Solomon. I would like to get in touch with you..."
+        class="messages" name="usrmessages"></textarea>
+      <input type="submit" value="Get it touch" class="touch-d" name="usrsubmit">
+    </div>
+  </form>
```

Comment:
- [ ] Great work building your form here 💯. However, please kindly ensure that you use only one `form` tag to wrap all your form fields, currently, you are using two different `form` tags for this job🤓.